.github/workflows/issue-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Labeler
+on:
+  issues:
+    types: [opened, edited]
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: github/issue-labeler@v3.1 
+      with:
+        configuration-path: .github/issue-labeler.yml
+        not-before: 2020-01-15T02:54:32Z
```

Comment:
Having "not-before" doesn't make sense.
It looks like you copied and pasted this from the example workflow.

Reply:
I would say that we probably don't want to backfill-label every issue.  We should set this to whatever UTC is for this (PR open)
![image](https://github.com/json-schema-org/website/assets/2676804/dc7c2834-bc29-4b1b-927e-2bbf535dce3f)

... or perhaps some date that we _do_ want to backfill with the label.